### DISCUSSION TITLE
検索機能でミュートやブロックしてる人の記事が続いていて表示ページに記事がなくてもヒット件数があるならページ戻し送りボタンを表示して欲しい

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#565] [要望・アイデア・不具合]
検索機能で、ヒット件数が沢山あってもミュートやブロックしてる人の記事が続いていると「一致する記事は見つかりませんでした。」と表示され、ページ戻し・送りボタンが表示されず次のヒット記事を見れなくなります。
しかし、URLのpageパラメタを指定すると記事が表示されます。
ヒット件数がある場合にはページ戻し送りボタンを表示していただけないでしょうか。

page=2 や page=3 ではページ戻し送りボタンなし。

![image](https://github.com/increments/qiita-discussions/assets/4151943/c3983f80-0647-40d4-b665-810f5f9023e4)

page=4 にすると記事タイトルが表示され、ページ戻し・送りボタンも表示されます。ヒットしたページは大量にあります。

![image](https://github.com/increments/qiita-discussions/assets/4151943/673a30ed-8ed2-486f-9b3c-66150f341da2)

**Comment by tanaka-asami415** [comment]
@shiracamus 
ご報告ありがとうございます🙇
修正できないか調査を行い検討します！

**Comment by kaitokimuraofficial** [comment]
@shiracamus

大変お待たせしました :bow:
検索結果の1ページ全てがミュートやブロック対象の記事であっても、
ページの戻し送りボタン(ページネーション)が表示されるようになりました。

また、ミュート対象の記事の表示方法についても、より良い改善策を検討していきます。

より良い検索体験を提供できるよう改善してまいりますので、ご期待ください。

**Comment by shiracamus** [comment]
確認できましたのでクローズいたします。
対応ありがとうございました。
